tools/LogViewer/nbproject/build-impl.xml

```diff
@@ -491,7 +491,7 @@ is divided into following sections:
     </target>
     <!--
                 =================
-                DEBUGGING SECTION
+                game.isDebugMode() SECTION
```

Comment:
this was probably not meant to be renamed :) also other .xml files

Reply:
🤦‍♂️